docs/contributing-to-airbyte/updating-documentation.md

```diff
@@ -20,3 +20,90 @@ If you're adding new files, don't forget to update `docs/SUMMARY.md`.
 
 Once you're satisfied with your changes just follow the regular PR process.
 
+## Connector Documentation Best Practices
+Connectors typically have the following documentation elements: 
+
+* READMEs
+* Changelogs
+* Github Issues & Pull Requests
+* Source code comments
+* How-to guides
+
+Below are some best practices related to each of these. 
+
+### READMEs
+Every module should have a README containing:
```

Comment:
Maybe a section on setting up the source/destination in the UI, explaining the fields?

Reply:
Agree. If this is just GH facing then I'm fine with that.